types/lib/model.d.ts

```diff
@@ -684,17 +684,7 @@ export interface Hookable {
 /**
  * Options for Model.findOrCreate method
  */
-export interface FindOrCreateOptions extends Logging, Transactionable {
-  /**
-   * A hash of search attributes.
-   */
-  where: WhereOptions;
-
-  /**
-   * Default values to use if building a new instance
-   */
-  defaults?: object;
-}
+export type FindOrCreateOptions = FindOptions & CreateOptions
```

Comment:
You still want to specify `defaults` option which is not covered by either `FindOptions` or `CreateOptions`

```js
types/test/model.ts:96:3 - error TS2345: Argument of type '{ where: { username: string; }; defaults: { beta_user: boolean; }; }' is not assignable to parameter of type 'FindOrCreateOptions'.

  Object literal may only specify known properties, and 'defaults' does not exist in type 'FindOrCreateOptions'.

96   defaults: {

     ~~~~~~~~~~~

97     beta_user: true

   ~~~~~~~~~~~~~~~~~~~

98   }

   ~~~
```